state-chain/pallets/cf-pools/src/lib.rs

```diff
@@ -837,12 +931,17 @@ pub mod pallet {
 		#[pallet::weight(T::WeightInfo::schedule_limit_order_update())]
 		pub fn schedule_limit_order_update(
 			origin: OriginFor<T>,
-			call: Box<Call<T>>,
-			dispatch_at: BlockNumberFor<T>,
-		) -> DispatchResultWithPostInfo {
+			base_asset: any::Asset,
+			quote_asset: any::Asset,
+			side: Side,
+			id: OrderId,
+			details: LimitOrderUpdateDetails<BlockNumberFor<T>>,
```

Comment:
Is the assumption here that `schedule_limit_order_update` isn't documented, and thus breaking compatibility isn'ta problem since it is only used internally in lp-api (which you also updated)?

Reply:
Also (as discussed), if `schedule_limit_order_update` isn't used externally, maybe we could just remove it and instead add distpatch_at field to `set_limit_order` and `update_limit_order` directly (as long is it is done in a backwards compatible way)? Wdyt @dandanlen?

Reply:
Any addition to the call parameters will break the SCALE encoding. The only way to make it fully backwards-compatible would be to add a new extrinsic and deprecate the old one. If we assume that everyone uses the LP API then we can assume it's a non-breaking change - probably a fair assumption? 

I do agree that adding a single param to the existing calls is probably better that a separate call (if nothing else, it makes the feature more visible).